android/app/src/main/res/values/styles.xml

```diff
@@ -1,20 +1,18 @@
 <resources>
-
     <!-- Base application theme. -->
     <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
         <!-- Customize your theme here. -->
         <item name="android:textColor">#000000</item>
         <item name="android:windowLightStatusBar">false</item>
-        <item name="android:statusBarColor">@color/launcher_background</item>
-        <item name="android:windowBackground">@color/launcher_background</item>
+        <item name="android:statusBarColor">@color/splash_background</item>
```

Comment:
This will set a white status bar, is that what we want?

Reply:
We might need 2 colors, `splash_background` & `status_bar`

Reply:
We left it as one there because the status bar color only shows up on that little spot between splash screen and app, but you're totally right we should separate those values. We hardcoded it whit because it's only for that portion since it is overridden by the application as soon it loads.

Reply:
@juancruzgs updated are we good to go?

Reply:
Does ReactNative override this value and sets something different for the rest of the app?
I'm asking because this is defined to be used in the entire application from the native side

Reply:
Yes, there are overrides per screen, each one will be responsible for setting the right color for the status bar.

Reply:
Okay I can remove these values later to avoid confusion
Looks good to me!